src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,8 +22,8 @@ public UserController(UserService userService,
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto findByEmail(@RequestParam String email) {
-        User user = userService.findByEmail(email).orElseThrow(
+    public UserResponseDto findByEmail(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).orElseThrow(
```

Comment:
im not sure whether you need to wrap email in authentication object here and then again retreive it back